server/src/projects/project-applicants/project-applicant.controller.ts

```diff
@@ -82,4 +86,25 @@ export class ProjectApplicantController {
        contact: newContact,
     }
   }
+
+  @Delete('/:id')
+  async deactivate(@Param('id') id) {
+    try {
+      await this.crmService.update('dcp_projectapplicants', id, {
+        statuscode: INACTIVE_STATUSCODE,
+        statecode: INACTIVE_STATECODE,
+      });
```

Comment:
ooh okay so we're not actually deleting the record, just deactivating it

Reply:
yeah, but it shouldn't work like this — the http verbs should match what they're actually doing.